maven-model-builder/src/main/java/org/apache/maven/model/interpolation/StringVisitorModelInterpolator.java

```diff
@@ -1424,7 +1424,7 @@ private void visit( Properties properties )
                     {
                         String value = (String) v;
                         String inter = interpolate( value );
-                        if ( value != inter )
+                        if ( value != inter && inter != null )
```

Comment:
Better to add logging for the interpolated value is null.

Reply:
But at most at debug level or even trace.

Reply:
I encounter this issue too, have to make use of maven 3.5 , failed on 3.6

Reply:
there is no need to log any - null is the normal value returned from `interpolate()` function